lib/MusicBrainz/Server/Form/Role/ValidateUsername.pm

```diff
@@ -0,0 +1,24 @@
+package MusicBrainz::Server::Form::Role::ValidateUsername;
+use HTML::FormHandler::Moose::Role;
+use MusicBrainz::Server::Translation qw( l );
+
+sub validate_username
+{
+    my ($self, $field) = @_;
+
+    my $username = $field->value;
+    if (defined $username) {
+        unless ($self->ctx->user->name == $username) { 
```

Comment:
The `validate_username` function that was in the Register form didn't have this check...does account creation still work? I expect it would either ISE or not run the validation.